src/main/java/com/iridium/iridiumskyblock/commands/AbstarctOpenInventory.java

```diff
@@ -0,0 +1,12 @@
+package com.iridium.iridiumskyblock.commands;
+
+import com.iridium.iridiumskyblock.database.Island;
+import org.bukkit.entity.Player;
+
+import java.util.Optional;
+
+public abstract class AbstarctOpenInventory {
+    boolean openPlayerInventory(Player player, Optional<Island> island){
```

Comment:
Why is this returning a boolean? and whats wrong with just returning an Inventory?